gns3/modules/vmware/dialogs/vmware_vm_wizard.py

```diff
@@ -47,7 +47,10 @@ def __init__(self, vmware_vms, parent):
             # we want to see the cancel button on OSX
             self.setOptions(QtWidgets.QWizard.NoDefaultButton)
 
-        if VMware.instance().settings()["use_local_server"]:
+            # Fusion is not supported on OSX
+            self.uiLocalRadioButton.setEnabled(False)
```

Comment:
I think this is a bug to leave this. Maybe we should explicitly show an error when on OSX?